content/docs/how-we-work/open-source-contribution/wordpress-contribution.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Wordpress Contribution
+weight: 3
+---
+
+# Wordpress Contribution
+
+## How to start Contributions
```

Comment:
@Yemaneberhan-Lemma, do you really need this heading? It's increasing the level of headings under it and I see there is no other top-level heading in the rest of the document.